contracts/external/DappHub/feed-factory.sol

```diff
@@ -0,0 +1,33 @@
+/// feed-factory.sol - deploy to create price-feeds
+
+// Copyright (C) 2017, 2018  DappHub, LLC
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity 0.4.25;
+
+import "./price-feed.sol";
+
+contract FeedFactory {
```

Comment:
Can we just deploy price feeds individually and authorize them in the medianizer instead of us the factory?

Reply:
Is the goal of importing these contracts just to replicate their set up for Brian to test or to deploy on our own?

Reply:
What would the difference be if we had to approve them in the medianizer? I want to change as little of the contracts as possible since those were audited and what is currently deployed